apps/financroo-tpp/clients.go

```diff
@@ -28,6 +28,7 @@ type Clients struct {
 	AcpPaymentsClient acpclient.Client
 	BankClient        BankClient
 	ConsentClient     ConsentClient
+	FDXConsentClient  FDXConsentClient
```

Comment:
Why this is necessary? BankClient is the interface that should be implemented for FDX